geemap/common.py

```diff
@@ -122,7 +124,7 @@ def ee_initialize(
                 ee.Authenticate(**auth_args)
                 ee.Initialize(**kwargs)
 
-        ee.data.setUserAgent(user_agent)
+    ee.data.setUserAgent(user_agent)
```

Comment:
Moved this outside the `if ee.data._credentials is None:` block so that we re-set the user agent string when there are multiple Maps (it is possible some are core and some are not).